recipes/dynamo-release/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "dynamo-release" %}
+{% set version = "1.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/aristoteleo/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4b9e0b9721802b0b7baa05a0651518958d512660703ca0d62a5140a7689f3d22
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - python >=3.7
+  host:
+    - python >=3.7
+    - numpy 1.23.*
+    - markdown 3.3.*
+    - setuptools
+    - pip
```

Comment:
Do you need all of these? Upstream only specifies python and nothing else?

Reply:
I think we need all of these in "host". I have removed the "build" section.